extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -578,7 +578,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
   };
 
   private getConvoRootEl = (anyInnerElement: HTMLElement) => {
-    return $(anyInnerElement).closest('div.if, td.Bu').first();
+    return $(anyInnerElement).closest('div.Bk, td.Bu').first();
```

Comment:
let's leave old class too, as it's possible that html code was updated not for all Gmail users, so we should keep older classes too

Reply:
Got it - I'll add it back. thank you!